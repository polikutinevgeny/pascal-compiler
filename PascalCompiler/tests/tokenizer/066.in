1..e.20