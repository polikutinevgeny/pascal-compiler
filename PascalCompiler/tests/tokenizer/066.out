Line |Pos  |Type        |Subtype                  |Value                              |Source                                            
----------------------------------------------------------------------------------------------------------------------------------------------
1    |1    |Constant    |IntegerConstant          |1                                  |1                                                 
1    |2    |Operator    |Range                    |..                                 |..                                                
1    |4    |Identifier  |Identifier               |e                                  |e                                                 
1    |5    |Operator    |Dot                      |.                                  |.                                                 
1    |6    |Constant    |IntegerConstant          |20                                 |20                                                
